app/views/html_publication/show.html.erb

```diff
@@ -34,11 +34,11 @@
 <%= render "govuk_publishing_components/components/inverse_header", {
   padding_top: 6,
   padding_bottom: 6,
-  subtext: content_item.last_changed,
+  subtext: @presenter.last_changed,
 } do %>
   <%= render "govuk_publishing_components/components/heading", {
     text: content_item.title,
-    context: I18n.t("content_item.schema_name.#{content_item.format_sub_type}", count: 1),
+    context: I18n.t("common.content_item.schema_name.#{@presenter.format_sub_type}", count: 1),
```

Comment:
I think this might need to be changed (in general I think we haven't explained what we're doing with locale keys, so this isn't on you).